resources/font.css

```diff
@@ -0,0 +1,131 @@
+/* Downloaded from https://fonts.googleapis.com/css?family=Titillium+Web:200,300,400,600,700 */
```

Comment:
Add a file header with copyright and licence

Reply:
Indeed, proceed like in the other CSS.

Reply:
Done